client/lib/plans/constants.js

```diff
@@ -410,11 +418,13 @@ export const PLANS_LIST = {
 			FEATURE_MALWARE_SCANNING_DAILY,
 		] ),
 		getSignupFeatures: () => compact( [
-			FEATURE_WORDADS_INSTANT,
 			FEATURE_MALWARE_SCANNING_DAILY,
+			FEATURE_VIDEO_CDN_LIMITIED,
+			FEATURE_WORDADS_INSTANT,
 			FEATURE_ALL_PERSONAL_FEATURES,
 		] ),
 		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' ),
+		getSignupBillingTimeFrame: () => 'per month', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? In the worst case we could lose several translated versions of it if we decide to not keep the test.